backbone.js

```diff
@@ -1511,13 +1535,18 @@
 
     // Convert a route string into a regular expression, suitable for matching
     // against the current location hash.
-    _routeToRegExp: function(route) {
+    _routeToRegExp: function(route, paramNames) {
```

Comment:
[This is what express uses](https://github.com/pillarjs/path-to-regexp), so passing the array is probably fine.